packages/ui/components/scheduler/components/grid/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+type Props = {
+  offsetHeight: number | undefined;
+  gridStopsPerDay: number;
+  children: React.ReactNode;
+  zIndex?: number;
+};
+
+export function SchedulerColumns({ offsetHeight, gridStopsPerDay, children, zIndex }: Props) {
+  return (
+    <ol
+      className="scheduler-grid-row-template col-start-1 col-end-2 row-start-1 grid auto-cols-auto sm:pr-8"
```

Comment:
`scheduler-grid-row-template` lives in styles.css and requires the data-gridstopsperday to function